<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telalogin</class>
 <widget class="QMainWindow" name="telalogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputUsuario">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSenha">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEntrar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
